### PR TITLE
Add webdriver.io tests and remove nightwatch tests. 

### PR DESCRIPTION
**Pull request by rakshith91**
### Summary

- Add wdio tests for no data view
- Remove nightwatch tests from `terra-clinical-error-view` and `terra-clinical-item-display`
- Remove nightwatch.conf.js 

Note: Not logging an issue for color-contrast violations in no data view since it's already deprecated. 

Thanks for contributing to Terra. 
@cerner/terra

[CONTRIBUTORS.md]: ../blob/master/CONTRIBUTORS.md
[License]: ../blob/master/LICENSE

### CODE REVIEW

**Comment by emilyrohrbough** [comment]
We probably shouldn't add wdio tests for error view. It is deprecated and we are not longer testing the component.

**Comment by rakshith91** [comment]
> No wdio tests were added to replace the item-display nightwatch tests.
@emilyrohrbough  item-display consumes wdio already. 
https://github.com/cerner/terra-clinical/blob/master/packages/terra-clinical-item-display/tests/wdio/item-display-spec.js

**Comment by rakshith91** [comment]
> We probably shouldn't add wdio tests for error view or no data view- they are both deprecated and we are not longer testing the component.

409a677addf83af862993afaa9940afa1362f4fd

File: packages/terra-clinical-no-data-view/tests/wdio/NoDataViewTestRoutes.jsx [hunk 1]
```diff
@@ -0,0 +1,21 @@
+/* eslint-disable import/no-extraneous-dependencies */
```
Comment (emilyrohrbough) [inline packages/terra-clinical-no-data-view/tests/wdio/NoDataViewTestRoutes.jsx]: This file needs removed

File: packages/terra-clinical-no-data-view/tests/wdio/NoDataViewTests.jsx [hunk 2]
```diff
@@ -0,0 +1,17 @@
+/* eslint-disable import/no-extraneous-dependencies */
```
Comment (emilyrohrbough) [inline packages/terra-clinical-no-data-view/tests/wdio/NoDataViewTests.jsx]: This file needs removed

File: generatedComponentConfig.js [hunk 3]
```diff
@@ -31,10 +31,10 @@ import NormalItemList from './packages/terra-clinical-detail-view/examples/test-
 import DefaultListItem from './packages/terra-clinical-detail-view/examples/test-examples/DetailListItem/DefaultListItem.example';
 import DetailViewDivided from './packages/terra-clinical-detail-view/examples/test-examples/DetailView/DetailViewDivided.example';
 import DetailViewNoDivider from './packages/terra-clinical-detail-view/examples/test-examples/DetailView/DetailViewNoDivider.example';
-import ButtonErrorView from './packages/terra-clinical-error-view/tests/nightwatch/ButtonErrorView';
```
Comment (emilyrohrbough) [inline generatedComponentConfig.js]: I know you already did this work, but I don't think we should move these file either. See conversation here https://github.com/cerner/terra-clinical/pull/296

File: generatedComponentConfig.js [hunk 4]
```diff
@@ -66,10 +66,10 @@ import DefaultLabelValueView from './packages/terra-clinical-label-value-view/ex
 import MultipleValueLabelValueView from './packages/terra-clinical-label-value-view/examples/test-examples/MultipleValueLabelValueView.example';
 import NodeValueLabelValueView from './packages/terra-clinical-label-value-view/examples/test-examples/NodeValueLabelValueView.example';
 import TextValueLabelValueView from './packages/terra-clinical-label-value-view/examples/test-examples/TextValueLabelValueView.example';
-import ContentNoDataView from './packages/terra-clinical-no-data-view/tests/nightwatch/ContentNoDataView';
-import DefaultNoDataView from './packages/terra-clinical-no-data-view/tests/nightwatch/DefaultNoDataView';
-import HiddenNoDataView from './packages/terra-clinical-no-data-view/tests/nightwatch/HiddenNoDataView';
-import TextNoDataView from './packages/terra-clinical-no-data-view/tests/nightwatch/TextNoDataView';
+import ContentNoDataView from './packages/terra-clinical-no-data-view/tests/wdio/ContentNoDataView';
```
Comment (emilyrohrbough) [inline generatedComponentConfig.js]: Again, I know you already did this work, but I don't think we should move these file either. See conversation here #296

Comment (rakshith91) [reply generatedComponentConfig.js]: https://github.com/cerner/terra-clinical/pull/309/commits/7891fe0d9d2e3e0fce9280e2f450124d39ee5463

**Pull request merged**